spec.html

```diff
@@ -33735,21 +33735,43 @@ <h1>String.prototype.lastIndexOf ( _searchString_ [ , _position_ ] )</h1>
+          <p>This method may rely on whatever language- and/or locale-sensitive comparison functionality is available to the ECMAScript environment from the host environment, and is intended to compare according to the conventions of the host environment's current locale. However, regardless of comparison capabilities, this method must recognize and honour canonical equivalence according to the Unicode Standard&mdash;for example, the following comparisons must all return `0`:</p>
+          <pre><code class="javascript">
+            // &#x212B; ANGSTROM SIGN vs.
+            // A&#x030A; LATIN CAPITAL LETTER A + COMBINING RING ABOVE
+            "\u212B".localeCompare("A\u030A")
+
+            // &#x2126; OHM SIGN vs.
+            // &#x03A9; GREEK CAPITAL LETTER OMEGA
+            "\u2126".localeCompare("\u03A9")
+
+            // &#x1E69; LATIN SMALL LETTER S WITH DOT BELOW AND DOT ABOVE vs.
+            // s&#x0307;&#x0323; LATIN SMALL LETTER S + COMBINING DOT ABOVE + COMBINING DOT BELOW
+            "\u1E69".localeCompare("s\u0307\u0323")
+
+            // &#x1E0B;&#x0323; LATIN SMALL LETTER D WITH DOT ABOVE + COMBINING DOT BELOW vs.
+            // &#x1E0D;&#x0307; LATIN SMALL LETTER D WITH DOT BELOW + COMBINING DOT ABOVE
+            "\u1E0B\u0323".localeCompare("\u1E0D\u0307")
+
+            // &#x1100;&#x1161; HANGUL CHOSEONG KIYEOK + HANGUL JUNGSEONG A
+            // &#xAC00; HANGUL SYLLABLE GA
+            "\u1100\u1161".localeCompare("\uAC00")
+          </code></pre>
+          <p>For a definition and discussion of canonical equivalence see the Unicode Standard, chapters 2 and 3, as well as Unicode Standard Annex #15, Unicode Normalization Forms (<a href="https://unicode.org/reports/tr15/">https://unicode.org/reports/tr15/</a>) and Unicode Technical Note #5, Canonical Equivalence in Applications (<a href="https://unicode.org/notes/tn5/">https://www.unicode.org/notes/tn5/</a>). Also see Unicode Technical Standard #10, Unicode Collation Algorithm (<a href="https://unicode.org/reports/tr10/">https://unicode.org/reports/tr10/</a>).</p>
```

Comment:
```suggestion
          <p>For a definition and discussion of canonical equivalence see the Unicode Standard, chapters 2 and 3, as well as <a href="https://unicode.org/reports/tr15/">Unicode Standard Annex #15, Unicode Normalization Forms</a> and <a href="https://unicode.org/notes/tn5/">Unicode Technical Note #5, Canonical Equivalence in Applications</a>. Also see <a href="https://unicode.org/reports/tr10/">Unicode Technical Standard #10, Unicode Collation Algorithm</a>.</p>
```

Link terms. Don't render URLs.